main/webapp/modules/core/scripts/facets/list-facet.js

```diff
@@ -161,7 +161,7 @@ ListFacet.prototype._initializeUI = function() {
           '<a href="javascript:{}" class="facet-choice-link" bind="resetButton">'+$.i18n('core-facets/reset')+'</a>' +
           '<a href="javascript:{}" class="facet-choice-link" bind="invertButton">'+$.i18n('core-facets/invert')+'</a>' +
           '<a href="javascript:{}" class="facet-choice-link" bind="changeButton">'+$.i18n('core-facets/change')+'</a>' +
-          '<span bind="titleSpan"></span>' +
+          '<span class="facet-title-span" bind="titleSpan" title="Click here to edit the name of the facet"></span>' +
```

Comment:
```suggestion
          '<span class="facet-title-span" bind="titleSpan" title="Click to rename facet"></span>' +
```

Reply:
Simple English for easier translations.  Can we also allow hover text to be translatable?